src/main/java/com/TableTalk/Enterprise/dao/RoomServiceDAOStub.java

```diff
@@ -1,41 +1,49 @@
-/*package com.TableTalk.Enterprise.dao;
+package com.TableTalk.Enterprise.dao;
 
```

Comment:
What is the point of having a stub if you're just going to comment it out. 

Technical suggestion: make your stub functional so you can properly test things out.